### PR TITLE
[3pt] Gremarketing migration

### PR DESCRIPTION
**Pull request by ashishpuliyel**
## What does this change?
Adding google reremarking to the commercial core

## Why?
Mostly to get used to the code base (https://trello.com/c/fSAE0Bof/1112-commjs-get-familiar-with-codebase)

### CODE REVIEW

**Review by mxdvl — COMMENTED** [review]
Looks good, but you may want to run `yarn prettier src --write` to fix the linting check :)

File: src/third-party-tags/remarketing.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { GetThirdPartyTag } from '../types';
+
+declare global {
```
Comment (ioanna0) [inline src/third-party-tags/remarketing.ts:3]: Could that be moved into `src/types.ts`? Also don't know if we could find and replace the type of google_trackConversion and google_tag_params from `any` to something more strictly typed?

**Comment by sndrs** [comment]
a little `yarn lint --fix` should clear up your linting woes

**Review by mxdvl — APPROVED** [review]

**Comment by ashishpuliyel** [comment]
I didn't see all your messages here, sorry! Useful tips I think I've now taken them all into account. Though there seem to be conflicts that I'll have to sort out now

**Comment by ashishpuliyel** [comment]
I'm not sure what i've done but I've borked the git history on this completely. WTF is this. Anyway. Monday I'm going to merge it on the command line.
Screenshot 2020-10-09 at 17 40 26

**Comment by mxdvl** [comment]
I managed to fix my issues by rebasing locally: `git rebase -i main`

**Pull request closed without merging**